floor/test/integration/view/view_test.dart

```diff
@@ -59,6 +60,24 @@ void main() {
         expect(actual, equals(expected));
       });
 
+// TODO this currently won't work as all `null` results won't be added to the Stream.
+// this is not specific to Views.
```

Comment:
I'm a bit torn on that one... I think that it can be reasonable to not return null in streams... But I think that it should be documented. I'll open an issue.